custom_components/netro_watering/translations/es.json

```diff
@@ -0,0 +1,130 @@
+{
+    "config": {
+        "abort": {
+            "already_configured": "Dispositivo ya configurado"
+        },
+        "error": {
+            "invalid_serial_number": "Número de serie incorrecto",
+            "unknown": "Error inesperado",
+            "unknown_device_type": "Tipo de dispositivo desconocido"
+        },
+        "step": {
+            "user": {
+                "data": {
+                    "device_name": "Nombre del dispositivo",
+                    "device_type": "Tipo del dispositivo",
+                    "serial_number": "Número de serie del dispositivo"
+                },
+                "description": "Por favor, introduzca el número de serie de su dispositivo",
+                "title": "Configure Netro"
```

Comment:
is it a spanish translation ? Same text as for english ?

Reply:
Yes, "Configure netro" is correct in Spanish as an imperative. Perhaps it would be more clear something like _Configura tu Netro_ or _Configuración de Netro_